Tests/ClientRuntimeTests/ClientRuntimeTests/NetworkingTests/ClientErrorTests.swift

```diff
@@ -1,147 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-import XCTest
-import AwsCommonRuntimeKit
-@testable import ClientRuntime
-
-class ClientErrorTests: XCTestCase {
```

Comment:
These tests are all for error equatability & for `WaiterTypedError` so they are all removed.